### PR TITLE
[XamlC] fix a bug in the debugging code introduced last week

### PR DESCRIPTION
**Pull request by StephaneDelcroix**
### Description of Change ###

Clean the drain pipes

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Build.Tasks/DebugXamlCTask.cs [hunk 1]
```diff
@@ -69,8 +69,8 @@ public override bool Execute()
 					}
 					var initCompRuntime = typeDef.Methods.FirstOrDefault(md => md.Name == "__InitComponentRuntime");
 					if (initCompRuntime == null) {
```
Comment (samhouts) [inline Xamarin.Forms.Build.Tasks/DebugXamlCTask.cs]: Bad curly brace position (for future fixing)

File: Xamarin.Forms.Build.Tasks/DebugXamlCTask.cs [hunk 2]
```diff
@@ -69,8 +69,8 @@ public override bool Execute()
...
-						LogLine(2, "no __InitComponentRuntime found... skipped.");
-						continue;
+						LogLine(2, "no __InitComponentRuntime found... duplicating.");
+						initCompRuntime = DuplicateMethodDef(typeDef, initComp, "__InitComponentRuntime");
```
Comment (samhouts) [inline Xamarin.Forms.Build.Tasks/DebugXamlCTask.cs]: Would it make sense to have "__InitComponentRuntime" be a const at some point?

**Comment by samhouts** [comment]
:+1:

File: Xamarin.Forms.Build.Tasks/DebugXamlCTask.cs (hunk 2 above)
Comment (StephaneDelcroix) [inline Xamarin.Forms.Build.Tasks/DebugXamlCTask.cs]: It would, at some point

**Comment by jassmith** [comment]
:+1:

**Pull request merged**